doc/lpvs-api.yaml

```diff
@@ -48,6 +48,26 @@ paths:
               schema:
                 $ref: '#/components/schemas/WebhookResponseForbidden'
 
+/:
+  post:
+    tags:
+      - Forwarding API
+    summary: Forward to Webhook
+    description: Forwards the request to the `/webhooks` endpoint.
+    requestBody:
+      content:
+        application/json:
+          schema:
+            type: object
+            description: The request body is forwarded as-is to the `/webhooks` endpoint.
+    responses:
+      '200':
+        description: Request successfully forwarded and processed by `/webhooks`.
+      '403':
+        description: Forbidden, invalid signature or access rights.
+      '500':
+        description: Server error while forwarding the request.
+
```

Comment:
Please apply my patch:

``````
diff --git a/doc/lpvs-api.yaml b/doc/lpvs-api.yaml
index 34b67c8..c2ed547 100644
--- a/doc/lpvs-api.yaml
+++ b/doc/lpvs-api.yaml
@@ -48,6 +48,31 @@ paths:
               schema:
                 $ref: '#/components/schemas/WebhookResponseForbidden'

+  /:
+    post:
+      tags:
+        - GitHub Webhooks API
+      summary: Forward to Start Scan
+      description: Forwards the request to the `/webhooks` endpoint.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/WebhookRequest'
+      responses:
+        '200':
+          description: 200 OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WebhookResponseOK'
+        '403':
+          description: 403 Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WebhookResponseForbidden'
+
   /scan/{gitHubOrg}/{gitHubRepo}/{prNumber}:
     post:
       tags:

``````